packages/account/package.json

```diff
@@ -66,13 +66,13 @@
         "history": "^5.0.0",
         "jest": "^26.4.2",
         "mini-css-extract-plugin": "^1.3.4",
-        "node-sass": "^4.12.0",
+        "node-sass": "^7.0.1",
```

Comment:
Does this work properly with node 14?

Reply:
I did not see any issues with the account package. Shall I add a note for QA to test account as well ?